### ISSUE TITLE
Pre-action checkboxes should point to the same location as the same action when it is your turn

### ISSUE THREAD

**Issue by bombizombi** [hippich/Bitcoin-Poker-Room#48]
I have no idea how anybody could miss this design idea.  In fact, I don't think there is a site in the world that does not have the checkboxes done in this way.

I'm talking about 'fold' out of turn checkboxes, and the situation that if controls change right before you click, you should still be doing the desired action, fold in this case.  So obvious.

**Comment by DamienBlack** [comment]
I agree. The placement of the check boxes and buttons should be reworked so that they are in the same place.
